spring-cloud-modules/spring-cloud-stream/spring-cloud-stream-kinesis/src/main/java/com/baeldung/kclkpl/KinesisKCLApplication.java

```diff
@@ -17,32 +18,31 @@ public class KinesisKCLApplication implements ApplicationRunner {
 
     @Value("${aws.access.key}")
     private String accessKey;
-    
+
     @Value("${aws.secret.key}")
     private String secretKey;
-    
+
     @Value("${ips.stream}")
     private String IPS_STREAM;
-    
+
     public static void main(String[] args) {
         SpringApplication.run(KinesisKCLApplication.class, args);
     }
 
-	@Override
-	public void run(ApplicationArguments args) throws Exception {
-		BasicAWSCredentials awsCredentials = new BasicAWSCredentials(accessKey, secretKey);
-		KinesisClientLibConfiguration consumerConfig = new KinesisClientLibConfiguration(
-		  "KinesisKCLConsumer", 
-		  IPS_STREAM,
-		  new AWSStaticCredentialsProvider(awsCredentials), 
-		  "KinesisKCLConsumer")
-		    .withRegionName(Regions.EU_CENTRAL_1.getName());
-
-		new Worker.Builder()
-		  .recordProcessorFactory(new IpProcessorFactory())
-		  .config(consumerConfig)
-		  .build()
-		  .run();
-	}
-    
+    @Override
+    public void run(ApplicationArguments args) throws Exception {
+        BasicAWSCredentials awsCredentials = new BasicAWSCredentials(accessKey, secretKey);
+
+        KinesisClientLibConfiguration consumerConfig = new KinesisClientLibConfiguration("KinesisKCLConsumer", IPS_STREAM, "", "", KinesisClientLibConfiguration.DEFAULT_INITIAL_POSITION_IN_STREAM, new AWSStaticCredentialsProvider(awsCredentials),
```

Comment:
@anastasiosioannidis please do a wild card static import for `KinesisClientLibConfiguration` so that we can directly use the constants here. It will also make the code more readable here as well as in the article.